index.html.slim

```diff
@@ -13,19 +13,19 @@ description: The simple, modern & productive way to build apps and infrastructur
       .spotlights-container
         a.spotlight-item href='https://developers.redhat.com/blog/?p=438176'
           img(src="https://static.jboss.org/images/rhd/promo/RHDP_homepromo_dotnetcore2announce_21aug2017" alt=".NET Core 2 Img")
-          span .NET Core for RHEL and the OpenShift Container Platform.
+          p .NET Core for RHEL and the OpenShift Container Platform.
```

Comment:
This should just be text instead of a block-level paragraph tag contained inside an inline anchor tag. Instead of putting the margin-bottom on the paragraph just target these classes of anchor tags. I think that will be more likely to eliminate the two-zone clicktale tracking.